### DISCUSSION TITLE
Active marker brightness on PWM mode

### DISCUSSION THREAD

**Discussion by stonkens** [bitcraze/discussions#1142] [Q&A]
We have an active marker deck that we want to use with a non-Qualysis motion capture system. When flashing firmware with PWM mode we do observe a difference when PWM is set to 0 vs 255. However, the LEDs are not bright at all and are not visible by the motion capture camera (or my phone for that matter) at more than 1m distance. Has anybody figured out how this occurs, it seems like the wattage shouldn't be an issue so it might be another setting that we have yet to modify

**Comment by knmcguire** [comment]
hi!

So you are saying that the firmware has been flashed with PWM mode, however this shouldn't be necessary. I believe that the active marker's LEDS should be able to be completely controlled by parameters, as described in the usage tab on the product page: https://www.bitcraze.io/products/active-marker-deck/

Here are the parameters that can be controlled from the parameter system, which also includes some more explanation: https://www.bitcraze.io/documentation/repository/crazyflie-firmware/master/api/params/#activemarker

If you don't find your answer there let us know!

**Reply by tobbeanton** [reply]
The IR-LEDs are SFH4716A that can go up to 1A (might get overheated at that current) but we have limited them to 40 mA since that is well enough for the qualisys system.

You could try to increase the current by changing the 68 ohm resistors that are located close to the LEDs. Voltage drop of LED is ~1.6V so at 3.7V battery you will get 2.1V over the resistor. Putting a 22 ohm resistor instead and you will get ~100 mA. I would say 250mA per LED would be the max to not stress the system. If you would have all four leds at 250 mA -> total 1A, almost 1/3 of the power would go to the LEDs when flying...

Other things that might affect the visibility is the wavelength, perhaps it doesn't match that well with a Optitrack system?
